### PR TITLE
"Meeting Link" and "Hosted By" text is hidden if empty

### PR DESCRIPTION
**Pull request by IanWearsHat**

### CODE REVIEW

File: apps/site/src/app/(main)/schedule/components/EventRegular.tsx [hunk 1]
```diff
@@ -93,13 +91,15 @@ export default function EventRegular({
...
+			)}
 			<p className="mb-2">
-				Hosted by:{" "}
-				{organization === undefined ? hosts?.join(", ") : organization}
-			</p>
-			<p className="mb-2">
-				{dateTimeFormat.formatRange(startTimeInPST, endTimeInPST)} PST |{" "}
-				<a href={virtual}>Meeting Link</a>
+				{dateTimeFormat.formatRange(startTimeInPST, endTimeInPST)} PST{" "}
+				{virtual && <a href={virtual}>| Meeting Link</a>}
```
Comment (waalbert) [inline apps/site/src/app/(main)/schedule/components/EventRegular.tsx:102]: After talking to log, we won't be putting Zoom or virtual links on the schedule page, so you can remove them instead. Sorry for this inconvenience

Comment (waalbert) [reply apps/site/src/app/(main)/schedule/components/EventRegular.tsx:102]: Oh wait we are using the same Zoom link for all.

Comment (IanWearsHat) [reply apps/site/src/app/(main)/schedule/components/EventRegular.tsx:102]: Oh so will this check be necessary or will we just have a hard-coded link

Comment (waalbert) [reply apps/site/src/app/(main)/schedule/components/EventRegular.tsx:102]: this check will be necessary

**Review by waalbert — APPROVED** [review]
Thanks for working on this!

**Review by samderanova — APPROVED** [review]

**Pull request merged**